examples/Workflow/WorkflowConsoleApp/Workflows/OrderProcessingWorkflow.cs

```diff
@@ -9,12 +9,34 @@ public class OrderProcessingWorkflow : Workflow<OrderPayload, OrderResult>
     {
         public override async Task<OrderResult> RunAsync(WorkflowContext context, OrderPayload order)
         {
+            var retryPolicy = Microsoft.DurableTask.TaskOptions.FromRetryPolicy(new Microsoft.DurableTask.RetryPolicy(
```

Comment:
This is not using Dapr's Retry policies, this is DurableTask retry policy from reading this. How does Dapr's retry policies fit here, since they should be consistent across all APIs including workflow.